script/start-odc.sh

```diff
@@ -66,11 +66,13 @@ function check_env_value_set() {
 function init_parameters() {
     echo "init parameters start"
 
-    # init parameters require environment variable
-    check_env_value_set DATABASE_HOST "${DATABASE_HOST}"
-    check_env_value_set DATABASE_PORT "${DATABASE_PORT}"
-    check_env_value_set DATABASE_USERNAME "${DATABASE_USERNAME}"
-    check_env_value_set DATABASE_NAME "${DATABASE_NAME}"
+    if [ -z "${ODC_BOOT_MODE}"  ] || [ "${ODC_BOOT_MODE}" != 'TASK_EXECUTOR' ]; then
```

Comment:
I think we should set these env event though we run odc as executor mode cause we may rely on metadb

Reply:
yes , ODC_BOOT_MODE will be passed by  k8s pod environment, check_env_value_set  will be skipped when odc start by ODC_BOOT_MODE

Reply:
revert this change